src/encoded/static/scss/bootstrap/_navbar.scss

```diff
@@ -48,7 +48,6 @@
 // content for the user's viewport.
 
 .navbar-collapse {
-  max-height: $navbar-collapse-max-height;
```

Comment:
Could this instead be overridden in, maybe, src/encoded/scss/encoded/modules/_navbar.scss? I’ve been considering the files in the bootstrap directory as non-modifiable since they come from the bootstrap team.

Reply:
Yes, see update. Thank you!